pl_bolts/models/self_supervised/simclr/transforms.py

```diff
@@ -129,8 +119,24 @@ def __init__(
         )
 
 
-@under_review()
 class SimCLRFinetuneTransform:
+    """Transforms for SimCLR during the fine-tuning stage.
+
+    Transform::
+
+        Resize(input_height + 10, interpolation=3)
+        transforms.CenterCrop(input_height),
+        transforms.ToTensor()
+
+    Example::
+
+        from pl_bolts.models.self_supervised.simclr.transforms import SimCLREvalDataTransform
+
+        transform = SimCLREvalDataTransform(input_height=32)
+        x = sample()
+        (_, _, xk) = transform(x)
+    """
+
```

Comment:
So, I'm not too versed in SimCLR, but either this docstring is wrong or the `__call__` is wrong since it definitely does not return 3-tuple.

Reply:
Thanks for spotting out this ❤️